examples/tweet/content.js

```diff
@@ -35,7 +36,7 @@ export class Content extends React.Component {
   }
   render() {
     let { media, text } = this.props
-    let styles = this.styles 
```

Comment:
Sorry for the noise, my editor (emacs) auto-magically removes trailing spaces...